ckanext/validation/utils.py

```diff
@@ -138,6 +141,22 @@ def run_sync_validation(resource_data):
         resource_data['_success_validation'] = True
 
 
+def delete_validation_report(resource_data):
+    resource_id = resource_data.get('id')
+    if not resource_id:
+        return
+
+    context = {u'ignore_auth': True}
+    data_dict = {u'resource_id': resource_id}
+
+    try:
```

Comment:
Wouldn't it make more sense to put this check inside the `resource_validation_delete` action?